### ISSUE TITLE
Is Aruba intended to support tool installation testing?

### ISSUE THREAD

**Issue by e2** [cucumber/aruba#377]
## Summary

According to the README.md, Aruba is designed "to make testing of command-line applications meaningful, easy and fun."

Does that include testing the **installation process** of command-line applications?

It is unclear whether that's within or beyond the scope of Aruba.

## Expected Behavior

Real life example: `tmuxinator` is a command-line tool packaged as a gem. Due to a recent RubyGems bug, it fails to install correctly. Basically, a symlink to the tool (`bin/mux`) is installed incorrectly, thus making it useless.

The only way to "catch" this bug is to: install the gem in an isolated environment and test it there.

I'd expect it to be clear whether this is an "exception" to what Aruba's README says or a "missing feature".

## Current Behavior

Aruba provides literally no support for testing tool installation in isolated environments. And current release plans suggest there never will be any such support included in Aruba.

## Possible Solution

At least clarify in the README whether a lack of such support is intended or if it's a "bug" or "omission".

## Steps to Reproduce (for bugs)

Try to create an Aruba test for testing `tmuxinator` after installation using the newest RubyGems from Ruby 2.3.0. 

Basically, this would mean a Cucumber scenario:

1. Installing `tmuxinator` from current sources (something currently not supported by Aruba)
2. Running the `mux` command and checking it's output

There is simply no example of how to achieve the first.

## Context

Without Aruba support, the only other options to test things like the `tmuxinator` `mux` command are:

1. Manually (by installing the gem and testing it) - which is no fun
2. Designing custom steps for this, even though they are the same for every single "tool gem" out there (which is no fun either)

So something Aruba promises to be fun is currently not fun at all.
